Packs/AWS-EKS/Integrations/AWSEKS/AWSEKS.py

```diff
@@ -0,0 +1,514 @@
+import json
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+from CommonServerUserPython import *  # noqa
+from AWSApiModule import *  # noqa: E402
+
+import urllib3
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
+
+''' HELPER FUNCTIONS '''
+
+
+def datetime_to_str(data: dict, param_name: str):
+    """
+    Update a datetime parameter to it's string value.
+    Args:
+        data: a dictionary with the response data
+        param_name: the name of the parameter that should be converted from datetime object to string.
+    """
+    if param_value := data.get(param_name):
+        data[param_name] = param_value.strftime(DATE_FORMAT)
+
+
+def validate_args(resources_vpc_config: dict, logging_arg: dict, authentication_mode: bool):
+    """
+    Check that exactly one argument is passed, and if not raise a value error
+    Args:
+        resources_vpc_config: An object representing the VPC configuration to use for an Amazon EKS cluster.
+        logging_arg: The cluster control plane logging configuration.
+        authentication_mode: The desired authentication mode for the cluster.
+
+    Returns:
+        A Command Results object
+    """
+    arr = [resources_vpc_config, logging_arg, authentication_mode]
+    arg_num = 0
+    for arg in arr:
+        arg_num = arg_num + 1 if arg else arg_num
+    if arg_num != 1:
+        raise ValueError("Please provide exactly one of the following arguments: resources_vpc_config, logging or "
+                         "authentication_mode.")
+    else:
+        return 'ok'
```

Comment:
```suggestion
    return True
```
Or we can remove it entirely